README.md

```diff
@@ -8,9 +9,11 @@ This repository will document different technology specific linting configuratio
 - [React with TypeScript](react-typescript-eslint.md)
 - [PHP v8.1 (Laravel v8) - PHPCS](php-laravel-phpcs.md)
 - [Node.Js v16 (NestJs v8)](node-nest-eslint.md)
+- [React Native v0.68](react-native-eslint.md)
```

Comment:
@vishnu-kyatannawar We should maintain some proper order here. Not in this PR necessarily but in general

Reply:
Okay sure